rinex/src/qc/analysis/obs.rs

```diff
@@ -243,34 +250,42 @@ fn report_ssi_statistics(
 }
 
 #[derive(Debug, Clone)]
+/// OBS RINEX specific QC analysis.  
+/// Full OBS RINEX analysis requires both the "obs" and "processing" features.
 pub struct QcObsAnalysis {
-    /// list of observables identified
+    /// Identified Observables
     observables: Vec<String>,
-    /// list of signals identified
+    /// Identified Signals
     signals: Vec<Carrier>,
-    /// list of codes encountered
+    /// Codes that were idenfitied
     codes: Vec<String>,
-    /// true if doppler observation is present
+    /// true if doppler observations were identified
     has_doppler: bool,
     /// Abnormal events, by chronological epochs
     anomalies: Vec<(Epoch, EpochFlag)>,
-    /// Total epochs
+    /// Total number of epochs identified
     total_epochs: usize,
     /// Epochs with at least 1 observation
     total_with_obs: usize,
     /// Complete epochs, with respect to given signal
     complete_epochs: Vec<(Carrier, usize)>,
+    #[cfg(feature = "obs")]
+    #[cfg_attr(docrs, doc(cfg(feature = "obs")))]
```

Comment:
These aren't public, so not sure if it makes sense to use `#[doc(cfg(feature = ...))]`.

Reply:
actually true. Now I prefer standard comments to document private attributes.

Just so you know, the current infrastructure contains structures to be dumped into HTML format.   
`to_html()` is public, it is used by the end user (`rinex-cli -qc` mode) and intends to expose all inner information that was estimated. I don't see any reasons to make those public, in the sense the end goal is to generate an analysis report.

Reply:
I meant they're not going to show up on docs.rs.